deploy/tests/schema/_posts.yml

```
@@ -0,0 +1,8 @@
+title: "String"
+config:
```

```suggestion
title: "String"

config:
```

Pedantic I know, but I think it helps clarify the fact that `title` is a constraint and `config` is general configuration, and they're not the same type of thing.